onedrive_service/src/onedrive_service/authentication/session.py

```diff
@@ -0,0 +1,33 @@
+import os
+from onedrivesdk import session
+
+
+class Session(session.Session):
+    """ OneDrive session representation.
+    """
+
+    def save_session(self, **save_session_kwargs):
+        """ Save the current session.
+        """
+
+        # TODO: Need to rewrite for safety.
+        super().save_session(**save_session_kwargs)
```

Comment:
It seems like this method duplicates the same method of the parent class - what is the sense in having it?

Reply:
This is just a stub class. We need to reimplement default Session functionality in feature for safety. 
Original class from SDK contains next comment:
`IMPORTANT: This implementation should only be used for debugging.
        For real applications, the Session object should be subclassed and
        both save_session() and load_session() should be overwritten using
        the client system's correct mechanism (keychain, database, etc.).
`

Reply:
But why `save_session()` is not overridden?

Reply:
`save_session` hasn't been overridden because we need to discuss in which way we will be saving session object in production. But for development it's enough to save session object in `pickle` object.

Reply:
Okay, I've got it.